ibis/backends/datafusion/__init__.py

```diff
@@ -622,10 +622,10 @@ def create_table(
         if schema is not None:
             schema = ibis.schema(schema)
 
-        properties = []
-
         if temp:
-            properties.append(sge.TemporaryProperty())
+            raise NotImplementedError(
+                "Datafusion does not support temporary tables on sql mode"
```

Comment:
```suggestion
                "DataFusion does not support temporary tables"
```

There's nothing other than SQL as far as we're concerned inside of Ibis, so pointing out SQL mode in the error message is a bit misleading in that it makes it seem like there's some other option 😅